test/jdk/java/foreign/TestHeapAlignment.java

```diff
@@ -45,7 +45,7 @@ public class TestHeapAlignment {
     public void testHeapAlignment(MemorySegment segment, int align, Object val, Object arr, ValueLayout layout, Function<Object, MemorySegment> segmentFactory) {
         assertAligned(align, layout, () -> layout.varHandle().get(segment, 0L));
         assertAligned(align, layout, () -> layout.varHandle().set(segment, 0L, val));
-        MemoryLayout seq = MemoryLayout.sequenceLayout(10, layout);
+        MemoryLayout seq = MemoryLayout.sequenceLayout(1, layout);
```

Comment:
This change was an actual issue in the test, which was made manifest by the new checks